container/README.md

```diff
@@ -0,0 +1,82 @@
+# ABRT Analytics in Podman container
+
+**Easiest way how to run ABRT Analytics (known historically as FAF)**
+---
+
+## How to deploy
+
+Prerequisites:
+A postgres database with semver extension is needed. Using abrt/postgres-semver
+image is recommended. Persistent storage is provided by means of a podman volume.
+A Redis container is required to run and schedule faf actions from the web UI (see below).
+Communication between ABRT Analytics, the database and Redis is ensured by running all three containers
+in the same podman pod.
+`podman volume create <volume_name>`
+`podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name <pod_name>`
+`podman run --pod <pod_name> -v /var/tmp/data:/var/lib/pgsql/data -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit postgres-semver`
+
+Running ABRT Analytics is as simple as:
+
+`podman run --pod <pod_name> --name faf -dit -v <volume_name>:/var/lib/pgsql/data -e PGHOST=localhost -e PGUSER=faf -e PGPASSWORD=scrt -e PGPORT=5432 -e PGDATABASE=faf faf-image`
```

Comment:
The volume shouldn't be mounted to the FAF container.

```suggestion
`podman run --pod <pod_name> --name faf -dit -e PGHOST=localhost -e PGUSER=faf -e PGPASSWORD=scrt -e PGPORT=5432 -e PGDATABASE=faf faf-image`
```

Reply:
Fixed.